packages/tabs/src/Tabs.js

```diff
@@ -75,6 +78,10 @@ class Tabs extends Component {
      * When variant is set to "canvas", the effective alignment will always be "left"
      */
     align: PropTypes.oneOf(AVAILABLE_ALIGNMENTS),
+    /**
```

Comment:
The component already allows for allowing the passing of a css `className` through using `...props` so we do not need to have it as an explicit prop here.

Reply:
I didn't add this prop, it is already being used, I just added it to the `propTypes` for increased type checking and documentation.

Reply:
Got it, but please still remove this as we do not don't usually to type checking for `className` and we it is in the `README` that users can apply `className` for styling.